pages/spotlight/index.js

```diff
@@ -0,0 +1,78 @@
+import { useRouter } from "next/router";
+import Header from "@/components/Header/Header";
+import Image from "next/image";
+import styled from "styled-components";
+import Navigation from "@/components/Navigation/Navigation";
+
+const StyledImage = styled(Image)`
+  border-radius: 8px;
+  position: relative;
+  top: -20px;
+  box-shadow: 0 6px 20px 5px rgba(0, 0, 0, 0.19);
+`;
+const StyledBookDetail = styled.article`
+  border: 2px solid black;
+  border-radius: 8px;
+  margin: 2rem auto;
+  text-align: center;
+`;
+const StyledSection = styled.section`
+  display: flex;
+  justify-content: space-around;
+  font-style: italic;
+`;
+
+const StyledDetailedPage = styled.div`
+  width: 90%;
+  max-width: 600px;
+  margin: auto;
+  position: relative;
+  padding-bottom: 50px;
+`;
+const StyledDescription = styled.p`
+  text-align: justify;
+  padding: 20px;
+  line-height: 1.3;
+  transition: height 1.4s;
+  word-wrap: break-word;
+`;
+
+const SytledBookStar = styled.h2`
+  text-align: center;
+  border: 1px solid black;
+  border-radius: 5px;
+  width: 200px;
+  margin: auto;
```

Comment:
I'm not sure whether margin auto has any effect, because the star got centered by text-align: center;  already. You can try to check if leaving it out changes anything.

Reply:
"Text-align" centers the title within the border.
In this case "margin: auto" centers the position of the whole element. Otherwise the element is aligned on the left side